packages/constellation/src/components/Compositions/Logo/Logo.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import { Logo, LogoComponent } from '.'
+
+export default {
+  argTypes: {
+    circle: {
+      control: { type: 'boolean' },
+    },
+    primary: {
+      control: { type: 'text' },
+    },
+    secondary: {
+      control: { type: 'text' },
+    },
+    secondaryVariant: {
+      control: { options: ['primary', 'badge'], type: 'radio' },
+    },
+  },
+  args: {
+    backgroundColor: 'var(--color-primary)',
+    circle: true,
+    primary: 'TEST',
+    secondaryVariant: 'primary',
+  },
+  component: Logo,
+  title: 'Compositions/Logo',
+} as ComponentMeta<LogoComponent>
+
+const Template: ComponentStory<LogoComponent> = (args) => {
+  return <Logo {...args} />
+}
+
+export const Primary = Template.bind({})
+
+const Image: ComponentStory<LogoComponent> = (args) => {
+  const image = (
+    <img
+      src='https://upload.wikimedia.org/wikipedia/commons/6/66/SMPTE_Color_Bars.svg'
+      alt='placeholder'
+      className='object-cover w-full h-full'
+    />
+  )
+
+  return <Logo {...args} primary={image} />
```

Comment:
It feels like requiring that the consumer pass a whole image element or other react node could result in some potential misuse/misadventures.

Have you considered using `imgSrc` and `fallbackText` props instead for the logo image and text? That way we can better control the styling of the image?

Although this may depend on Blockchain.com's specific requirements. @blockdylanb are you aware of any specific requirements for how to pass images to these components.

Reply:
Yeah you have a very good point, possibly a safer option would be an imgSrc and I should add icons an as option there too